ios/FluentUI.xcodeproj/project.pbxproj

```diff
@@ -1345,7 +1345,7 @@
 			outputFileListPaths = (
 			);
 			outputPaths = (
-				"${target_dir}/FluentUI-Swift.h",
+				"${BUILT_PRODUCTS_DIR}/include/${PRODUCT_MODULE_NAME}/FluentUI-Swift.h",
```

Comment:
I wonder if this might also be part of the issue - I think it's supposed to be $(<setting name>) instead of ${<setting name>} (parentheses instead of curly braces)?